src/components/CommandBar/CommandBar.tsx

```diff
@@ -149,6 +151,7 @@ export class CommandBar extends React.Component<ICommandBarProps, ICommandBarSta
                          data-command-key={ index }
                          aria-haspopup={ !!(item.items && item.items.length) }
                          role='menuitem'
+                         aria-label={ `${ item.name }. ${ index === 0 ? this.props.itemNavigationScreenReaderText : '' }` }
```

Comment:
They would derive this.props.itemNavigationScreenReaderText to have their own navigation text, it only apples to the first item in the menu. I prepend name their, because otherwise it wouldn't read the name of the command